### ISSUE TITLE
Support switchbot curtains

### ISSUE THREAD

**Issue by daenny** [fphammerle/switchbot-mqtt#23]
I updated the switchbot library to support curtains https://github.com/Danielhiversen/pySwitchbot/pull/13

Can you implement the changes here as well? Otherwise, I might try in the weekend, if I find the time.

**Comment by fphammerle** [comment]
Thanks for adding curtain support to `pySwitchbot`!

We need a new `pySwitchbot` release ( https://pypi.org/project/PySwitchbot/#history ) before we can adapt `switchbot-mqtt`.

**Comment by fphammerle** [comment]
`pySwitchbot==0.9.0` was just released, I'll take a look

**Comment by fphammerle** [comment]
@daenny, I just pushed a pre-release with curtain support: https://pypi.org/project/switchbot-mqtt/0.6.0a0/

Could you check if that works for you?

**Comment by daenny** [comment]
Just tested via direct sending topics:
`mosquitto_pub -h 192.168.2.112 -u xxx-P xxxxx! -t homeassistant/cover/switchbot-curtain/xxxx/set -m OPEN`


```
2020-12-19T13:13:25+0000:DEBUG:switchbot:Sending command to switchbot 570f450105ff00
2020-12-19T13:13:25+0000:DEBUG:switchbot:Connecting to Switchbot...
2020-12-19T13:13:26+0000:DEBUG:switchbot:Connected to Switchbot.
2020-12-19T13:13:26+0000:DEBUG:switchbot:Prepare to send
2020-12-19T13:13:26+0000:DEBUG:switchbot:Sending command, 570f450105ff00
2020-12-19T13:13:26+0000:INFO:switchbot:Successfully sent command to Switchbot (MAC: xxxx).
2020-12-19T13:13:26+0000:DEBUG:switchbot:Disconnecting
2020-12-19T13:13:26+0000:INFO:switchbot_mqtt:switchbot curtain xxxxxxx opening
2020-12-19T13:13:26+0000:DEBUG:switchbot_mqtt:publishing topic=homeassistant/cover/switchbot-curtain/xxx/state payload=b'opening'
```

And the curtains move 👍 !

Open/Stop/Close commands all working thanks! 

A shortcoming about the `pySwitchbot` is that there is no feedback about the state. So I wonder if the reported state should be for now "Open" instead of "Opening". I will integrate in homassistant now and check the behaviour there.

**Comment by daenny** [comment]
```
cover:
  - platform: mqtt
    name: "Curtains"
    command_topic: "homeassistant/cover/switchbot-curtain/xx:xx:xx:xx/set"
    state_topic: "homeassistant/cover/switchbot-curtain/xx:xx:xx:xx/state"
```

It works fine, the state is indeed shown as opening, but that's fine for now. Thanks!

**Comment by fphammerle** [comment]
Great, thanks for testing!

I released the new version as `v0.6.0`.

Please open a new issue or PR, if you need opportunistic states (`open` instead of `opening)
